src/obfl-specification.html

```diff
@@ -2759,6 +2760,25 @@ <h4 id="L21851">Example</h4>
   </tbody>
 </table>
   
+<h3 id="L2762">mark-capital-letters</h3>
+
+<p>The overall policy for marking capital letters should be determined by the
```

Comment:
Can you please be more to the point? Please mention the purpose of this element clearly, that is to produce text without captials.